### PR TITLE
Fixes #868: Make imagemin non-default task.

### LINKED ISSUE #811: CONTRIBUTING.md: explain what "build" does and when you might need to build the project
I have the impression that changing certain HTML files requires "build"ing before you test the changes locally, but I have actually no clear picture of what building does and when one might need to use the build command. More details would be nice.

### LINKED ISSUE #868: Make imagemin a non-default grunt task
Spun out from #839.

### PR DESCRIPTION
**Pull request by miketaylr**
Also fixes #811 by documenting what build tasks do.

r? @hallvors

### CODE REVIEW

**Comment by miketaylr** [comment]
Ping @hallvors :)

**Comment by magsout** [comment]
:+1:

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -333,7 +333,14 @@ make start
 
 You should now have a local instance of the site running at `http://localhost:5000/`. Please [file bugs](https://github.com/webcompat/webcompat.com/issues/new) if something went wrong!
 
-### Building Project
+### Building the Project
+
+You need to build the project before serving it from a webserver will work, and after certain kinds of changes are made.
+
+* CSS: a build will run cssnext, combine custom media queries, run cssnext, and concat all source files into webcompat.dev.css. You'll need to re-build the CSS to see any changes, so it's recommended to use a watch task (see `make watch` or `grunt watch`).
```
Comment (hallvors) [inline CONTRIBUTING.md]: Will cssnext  run twice or is it a typo?

**Comment by hallvors** [comment]
Ops, sorry. :+1: with the small comment above addressed :)

File: CONTRIBUTING.md (hunk 1 above)
Comment (miketaylr) [inline CONTRIBUTING.md]: Ha, oops.

**Comment by miketaylr** [comment]
Thanks!

**Pull request merged**